yoyodyne/models/lstm.py

```diff
@@ -280,13 +272,19 @@ def beam_decode(
                 break
         # Returns the top-n hypotheses.
         histories = heapq.nlargest(n, hypotheses)
-        predictions = torch.tensor([h[1] for h in histories], self.device)
+
+        # Sometimes path lenghts does not match so it is neccesary to pad it
+        # all to same lenght to create a tensor.
+        max_len = max(len(h[1]) for h in histories)
+        predictions = torch.tensor(
+            [h[1] + [special.PAD_IDX] * (max_len - len(h[1])) for h in histories], device=self.device)
```

Comment:
I think we normally put the closing parenthesis on a new line here -- @kylebgorman you are more the code formatting person though :).

Reply:
For this, at the top-level directory:

```
pip install -r requirements.txt
black -l79 .
```